services/galley/src/Galley/API/Update.hs

```diff
@@ -476,3 +536,5 @@ checkOtrRecipients usr sid prs vms vcs val now
         OtrReportMissing us -> Clients.filter (`Set.member` us) miss
         OtrIgnoreMissing us -> Clients.filter (`Set.notMember` us) miss
 
+selfConv :: UserId -> ConvId
```

Comment:
👍 
Perhaps we can reuse that [here](https://github.com/wireapp/wire-server/blob/develop/services/galley/src/Galley/Data.hs#L334) after to make it clearer